test/e2e-tests.sh

```diff
@@ -36,7 +36,7 @@ go_test_e2e -timeout=20m ./test/... || failed=1
 # Run these _after_ the integration tests b/c they don't quite work all the way
 # and they cause a lot of noise in the logs, making it harder to debug integration
 # test failures.
-${REPO_ROOT_DIR}/test/e2e-tests-yaml.sh --run-tests || failed=1
+go_test_e2e -tags='e2e,examples' -timeout=20m -test.run '^TestExamples' ./test/ || failed=1
```

Comment:
So `go test -tags e2e` only run go tests, but `go test -tags e2e,examples` run both, hence the `-test.run '^TestExamples`. I wonder if having a build tag make sense (if we use the run filter anyway)

Reply:
I've managed to use only an `examples` build tag, so I've removed the `-test.run`